modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -596,7 +602,8 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     } catch (err) {
       hash = createKeccakHash('keccak256') as Hash;
     }
-    const hashBuffer = hash.update(Buffer.from(txToSign, 'hex')).digest();
+    // check what the encoding is supposed to be for message
```

Comment:
Not sure what this comment is referring to

Reply:
Didn't we say that message aren't hex encoded?  So wouldn't we need to buffer from some other encoding scheme?